devops/openshift/workers-deploy.yml

```diff
@@ -178,3 +180,5 @@ parameters:
     value: redis-creds
   - name: REDIS_PASSWORD_KEY
     value: password
+  - name: PORT
```

Comment:
do we need specific about the port name, in order to avoid confusion in the future?